share/completions/gem.fish

```diff
@@ -99,10 +99,8 @@ complete $install_opt -s l -l local -d "Restrict operations to the LOCAL domain
 complete $install_opt -s r -l remote -d "Restrict operations to the REMOTE domain"
 complete $install_opt -s b -l both -d "Allow LOCAL and REMOTE operations"
 complete $install_opt -s i -l install-dir -d "Gem repository directory to get installed gems" -x
-complete $install_opt -s d -l rdoc -d "Generate RDoc documentation for the gem on install"
-complete $install_opt -l no-rdoc -d "Don't generate RDoc documentation for the gem on install"
-complete $install_opt -l ri -d "Generate RI documentation for the gem on install"
-complete $install_opt -l no-ri -d "Don't generate RI documentation for the gem on install"
+complete $install_opt -s N -l no-document -d "Disable documentation generation on install"
+complete $install_opt -l document -d "Specify the documentation types you wish to generate. e.g., rdoc,ri"
```

Comment:
Does this take "rdoc" and "ri" as an argument?

In that case, it should be offered, e.g.

```fish
complete $install_opt -l document -a 'rdoc ri'
```

or, if it takes it as a comma-separated list, `__fish_append , rdoc ri`.

Reply:
Thanks. Argument will be `--document=rdoc` or `--document=rdoc,ri` so I took the latter in  906ebae